spec/hand_rankings/find_pair_spec.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../../lib/poker_hands/hand_rankings/find_pair'
+require_relative '../../lib/poker_hands/card'
+require 'pry'
```

Comment:
We should be requiring our `spec_helper` here instead of `pry`

Reply:
sounds good